### ISSUE TITLE
Dracula editor theme not considered dark

### ISSUE THREAD

**Issue by s-fleck** [rstudio/rstudio#2020]
### System details

    OS Version      :  Ubuntu 16.04
    RStudio Version : Rstudio Server Professional 1.1.383.
    R Version       :

### Steps to reproduce the problem

* Select dracula as your editor theme
* `rstudioapi::getThemeInfo()`

### Describe the problem in detail

Dracula is a dark theme, but `rstudioapi::getThemeInfo()$dark` is `FALSE`

```
$editor
[1] "Dracula"

$global
[1] "Modern"

$dark
[1] FALSE
```

**Comment by jmcphers** [comment]
Thanks for reporting! This is fixed in 1.2 via https://github.com/rstudio/rstudio/commit/a66965cffa69e831627b8794bdbaaeeb9e0119fe#diff-ede85aaa4cad3424f4cae655c5408f80.

**Comment by dfalty** [comment]
Verified fixed 1.2.359.
